From 97b9bc22cf099547fb4084e402fdd0513dedb71c Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Tue, 24 Jul 2018 19:53:54 +0200
Subject: [PATCH 212/249] wiki: Use custom TWRP for pioneer

* Since payload.bin is no longer test-keys signed
  for time being we are forced to repack official
  TWRP images with our own public key.

Change-Id: I81ba768686bc8c7f9fa3ea2b39070edfca4d2c47
---
 _data/devices/pioneer.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/_data/devices/pioneer.yml b/_data/devices/pioneer.yml
index 808558c..6094bd1 100644
--- a/_data/devices/pioneer.yml
+++ b/_data/devices/pioneer.yml
@@ -11,6 +11,7 @@ cpu: Cortex-A53
 cpu_cores: '8'
 cpu_freq: 2.2 GHz
 current_branch: 15.1
+custom_twrp_link: https://www.androidfilehost.com/?fid=5862345805528057397
 depth: 9.7 mm (0.38 in)
 download_boot: With the device powered off, hold <kbd>Volume Up</kbd> and connect the USB cable. The notification light should turn blue.
 gpu: Adreno 508
-- 
2.17.1

